client/package.json

```diff
@@ -51,8 +51,8 @@
     "@ngx-gallery/gallerize": "4.0.3",
     "@ngx-gallery/lightbox": "4.0.3",
     "@node-steam/vdf": "2.1.0",
-    "@sentry/angular": "^7.11.1",
-    "@sentry/tracing": "^7.11.1",
+    "@sentry/angular": "7.11.1",
+    "@sentry/tracing": "7.11.1",
```

Comment:
This is in a separate commit for some reason, please fixup into the other and force-push (we always make sure to keep the master branch all of Momentum repo's as clean as possible, even if that requires other branches are regularly force-pushed to)

Reply:
Done